pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/Codec.java

```diff
@@ -84,4 +87,24 @@
      * @throws IOException If it is impossible to read from the {@link ReadableSequentialData}
      */
     boolean fastEquals(@NonNull T item, @NonNull ReadableSequentialData input) throws IOException;
+
+    /**
+     * Converts a Record into a Bytes object
+     *
+     * @param item The input model data to convert into a Bytes object.
+     * @return The new Bytes object.
+     * @throws RuntimeException wrapping an IOException If it is impossible
+     * to write to the {@link WritableStreamingData}
+     */
+    default Bytes toBytes(@NonNull T item) {
+        byte[] bytes;
+        try (ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+             WritableStreamingData writableStreamingData = new WritableStreamingData(byteArrayOutputStream)) {
+            write(item, writableStreamingData);
+            bytes = byteArrayOutputStream.toByteArray();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
Can be a UncheckedIOException

Reply:
Fixed.

Reply:
Getting the following error using UncheckedIOException. Keeping the IOException...
/Users/user/local/pbj/pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/Codec.java:102: error: unreported exception IOException; must be caught or declared to be thrown
        try (ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
                                   ^
  exception thrown from implicit call to close() on resource variable 'byteArrayOutputStream'

Reply:
oops I was wrong

Reply:
Does it need to catch Exception, what is actually thrown. Not just IOExceptions?

Reply:
Yes. It seems the close() method on WritableStreamData throws it. That might be a bug, though.
Here is the error message...
/Users/user/local/pbj/pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/Codec.java:102: error: unreported exception Exception; must be caught or declared to be thrown
             WritableStreamingData writableStreamingData = new WritableStreamingData(byteArrayOutputStream)) {
                                   ^
  exception thrown from implicit call to close() on resource variable 'writableStreamingData'

Reply:
I think it is a bug. I changed the closew() method to throw IOException.

Reply:
Now that it is a `IOException` then the runtime can be `UncheckedIOException`